host_modules/docker_service.py

```diff
@@ -44,6 +44,23 @@ def is_allowed_container(container):
     return False
 
 
+def is_allowed_image(image):
+    """
+    Check if the image is allowed to be managed by this service.
+
+    Args:
+        image (str): The image name.
+
+    Returns:
+        bool: True if the image is allowed, False otherwise.
+    """
+    image_name = image.split(":")[0]  # Remove tag if present
+    for allowed_image, _ in ALLOWED_CONTAINERS:
```

Comment:
Thanks @hdwhdw for the work! Could you add more description for the PR? Seems we need to maintain the container list, which I have one new container "docker-sonic-bmp" added.

Reply:
could you help to contain bmp first? pls refer below
![image](https://github.com/user-attachments/assets/55737924-c57b-43c7-8158-39c9530c373e)

Reply:
Done.